core/src/memory.rs

```diff
@@ -205,7 +174,7 @@ pub fn pages_to_wasm_pages_set<'a>(
                 wasm_pages.insert(WasmPageNumber(gp.0 / PageNumber::num_in_one_wasm_page()));
                 Ok(())
             } else {
-                Err("There is wasm page, which has not all gear pages in the begin")
+                Err(MemoryError::NotAllPagesInEnd)
```

Comment:
Maybe something like `NotAllPagesInBegin` should be used? The error message states `... in the begin`